macros/stitch_user_identifiers.sql

```diff
@@ -4,21 +4,34 @@
 
 {% macro default__stitch_user_identifiers(enabled, relation=this, user_mapping_relation='snowplow_web_user_mapping') %}
     {% if enabled | as_bool() %}
-      -- Update sessions table with mapping
+
+      {% if var('snowplow__page_view_stitching') %}
+        alter table if exists {{ this }} add column if not exists stitched_user_id {{ type_string() }};
+      {% endif %}
+
```

Comment:
Why not do what we do in sessions and just add it in the model?

Reply:
Didn't we agree that this will be optional and we don't want the field to be there as null for 99% of the users?

Reply:
Probably, although it's optional for sessions, we just populate it with the domain_userid if it's not enabled - not sure if it's better for consistency...I'm fine either way, but I wasn't aware of this syntax working for all warehouses?